src/test/java/seedu/address/logic/parser/TimeParserTest.java

```diff
@@ -37,6 +37,97 @@ void testInvalidDate() {
         assertTrue(hasError);
     }
 
+    @Test
+    void testDayAnd24hTime() throws ParseException {
+        TimeParser.parseDate("Tue 1630");
```

Comment:
Perhaps the test method name here could be something along the lines of `parseDate_validDayAnd24hTime_successfulParse`

Reply:
Thanks for the heads up! have edited the names of the tests that I am intending to keep! 👍